src/Components/App/App.js

```diff
@@ -0,0 +1,22 @@
+import './App.scss';
+import {Homepage} from '../HomePage/Homepage'
```

Comment:
We should consider making this consist from the get-go.  Do we want `export default` syntax for our components, or just to export them directly upon definition?  I think it's easier to just `export default` but that's just my suggestion.

Reply:
I prefer to export a single element when possible, especially when exporting components meaning something like this `export const Component = () => {}`